### ISSUE TITLE
Don't show the accuracy of test set

### ISSUE THREAD

**Issue by tzutalin** [NVIDIA/DIGITS#190]
Hi,
   I create a dataset containing train, val, and test set. While training, the UI will only show the accuracy on validation set. However, would DIGITS be possible to show the accuracy of test set and train set at the same time? For instance, if we can know the accuracy about train and validation, we can know whether our model is overfitting or underfit. 
   If we specify a model with the best accuracy on validation set, is DIGITS possible to use the model to test the accuracy on test set?  

Regards

**Comment by lukeyeager** [comment]
To computer the accuracy over the training set, you can simply customize the network and remove [these lines](https://github.com/NVIDIA/DIGITS/blob/v2.0.0-rc3/digits/standard-networks/alexnet.prototxt#L366-L368) from your `Accuracy` layer:
```proto
include {
  phase: TEST
}
```
NOTE: Caffe uses `TEST` to refer to the Validation phase - see https://github.com/BVLC/caffe/issues/1245#issuecomment-74999077.

The `Test` set that you can create in DIGITS is really just a placeholder for future functionality like #137. You can't do anything useful with it at the moment.

**Comment by tzutalin** [comment]
Thanks for replying. It seems https://github.com/NVIDIA/DIGITS/pull/137 can work my problem out.

**Comment by sameerkhann** [comment]
validation loss 0.06 and validation accuracy 97.7  does not make sense because these two values doesnt compliment each other

**Comment by gheinrich** [comment]
@sameerkhann: the logistic loss does not measure the proportion of wrong classifications. You might want to review the formulation [there](http://caffe.berkeleyvision.org/doxygen/classcaffe_1_1SoftmaxWithLossLayer.html#details). In practice `loss+accuracy` is almost never equal to `100`.
